notes/roman-urdu/103_Surah_Asar.md

```diff
@@ -0,0 +1,16 @@
+Number 103: Surah Asar
+Ayahs: 3
+Rukus: 1
+Related Surah: [104. Surah Humaza](notes/roman-urdu/104_Surah_Humaza.md)
+
+### Zamane ki qasam Insan bare khasare m h
+
+### Sawai wo log jo in 4 kaam poore karta ho
+1. Jinho ne imaan lana.
+2. Naik amal
+3. Haq ki naseehat.
+4. Sabar ki talqain.
+5. Allah par uske rasool malaik aakhrat par imaan la kar phr Allah k batae huey un baato par amal kre. na sirf khud balqai wo haq/islam ki baat dusro tak pohchaye ... jb yeh kam kre ga to museebatein aaye gi. unka sabar se kaam le.
```

Comment:
Nits:
- 4 kaam alag or final description alag alag honay chahiyein.
- Allah par, uskay rasool par, malaika par aur akhirat par imaan la kar phir Allah k batai hoi baato par amal kre. Na sirf khud balqai wo haq/islam ki baat dusro tak pohchaye ... jb yeh kam kre ga to museebatein aaye gi. Tab ja k sabar se kaam le.